dist/bin/current/helperScripts/resolveAppConfig.sh

```diff
@@ -51,7 +51,12 @@ function setRoddyBinaryVariables() {
   fi
   local numberOfDots=$(grep -o "[.]" <<< "$useRoddyVersion" | wc -l)    # Try find out, if we got an api level input like 2.2 or 2.3
   if [[ $numberOfDots == 1 ]]; then
-    useRoddyVersion=$(cd $RODDY_DIRECTORY/dist/bin; ls -d $useRoddyVersion* | sort -V | tail -n 1)
+    useRoddyVersionDir=$(cd $RODDY_DIRECTORY/dist/bin; find -maxdepth 1 -name "$useRoddyVersion*")
```

Comment:
I guess the * is to get the latest entry when using the short syntax? So e.g. 2.3 => 2.3.112?

But how about directory order returned by find?

Reply:
The directory order is fixed to version order below.